### ISSUE TITLE
Update Create quiz sections Interface and Add New Features

### ISSUE THREAD

**Issue by Boobeshkumar56** [vishalverma9572/QuizMaster#5]
I want to improve the create section of our quiz app. The updates will include a new, user-friendly design, the ability to select multiple answers(can apply this option for a paticular question) for questions using checkboxes, and an option to assign point values to each question. These changes will make it easier for users to create quizzes and allow for more flexible scoring.

**Comment by Boobeshkumar56** [comment]
I would like to work on this project,  could you please assign it under #hacktoberfest, that'd be great

**Comment by vishalverma9572** [comment]
@Boobeshkumar56 work on this issue and create PR after a proper testing...

**Comment by contributeoss2024** [comment]
Hi, I'd like to contribute to this project.

**Comment by vishalverma9572** [comment]
@contributeoss2024 , of course! Please tell me how you're planning to contribute to resolving this issue.

**Comment by contributeoss2024** [comment]
As a new UX design contributor, I will begin by researching to propose new features. A competitor review will help with this process. Next, I will address interface issues, focusing on design principles such as visibility, consistency, and style. Finally, I will create wireframes and provide everything in a Figma file for the developer to use

**Comment by contributeoss2024** [comment]
Can you assign me to that project?

**Comment by vishalverma9572** [comment]
@contributeoss2024, , assigned! Please work on this in small steps to make maintenance easier. Let me know which part you're starting with first

**Comment by contributeoss2024** [comment]
I am sorry, I removed my assignment by mistake. Could you assign it to me again? I have done some research and found some features that we can add to our quiz app. Should I send them as a pull request or as a comment here? 
thank you

**Comment by vishalverma9572** [comment]
@contributeoss2024 , Do you want to continue working or what?

**Comment by vishalverma9572** [comment]
@Boobeshkumar56 , any updates?

**Comment by Boobeshkumar56** [comment]
i am currently working on this update,i will update it today
